### ISSUE TITLE
Gridded products

### ISSUE THREAD

**Issue by isedwards** [opencdms/Climsoft#458]

**Comment by Patowhiz** [comment]
@smachua will this require major changes to the data model, can the current model as it is ingest this type of data? 

It's currently indicated as a feature release of version 4.x in the [road map]https://github.com/opencdms/Climsoft/wiki/Climsoft-Roadmap) .

**Comment by isedwards** [comment]
The roadmap on the Climsoft wiki was last updated in January 2017.  It will be replaced soon by the new roadmap that was started last week.  The new roadmap will give more details of future support for gridded data.

**Comment by Patowhiz** [comment]
Okay. We will set it's milestone once that road map is ready.

**Comment by smachua** [comment]
@Patowhiz the current data model cannot accommodate Gridded Data. This is not station data but generated data that is identified by its geographical location. Those locations are not fixed hence cannot be accommodated in the station metadata.

**Comment by Patowhiz** [comment]
So then I presume this will be a major advancement in in the data model and will have to wait till version 5

**Comment by mhabimana** [comment]
@Patowhiz :Yes, It makes sense.

**Comment by smachua** [comment]
@Patowhiz yes just leave it out for now. We're not even sure whether V5 data model will accommodate it. There are discussions in other circles regarding archival/retrieval of Gridded data.

**Comment by isedwards** [comment]
The new road map currently suggests that gridded data will be prototyped during version 6.x, aiming for stable release in version 7.  I agree that it's best just to leave the [future](https://github.com/opencdms/Climsoft/labels/future) label for now and schedule this when it gets funded.
